README.md

```diff
@@ -76,3 +76,17 @@ flask run
 ## 📜 LICENSE
 
 [![GitHub license](https://img.shields.io/github/license/kritikaparmar-programmer/HealthCheck?logo=github)](https://github.com/kritikaparmar-programmer/HealthCheck/blob/main/LICENSE)
+
+## Contributor:

```

Comment:
Remove this Contributor heading and let it be just Credit one.

Reply:
Done !! Have a look